azurerm/data_source_log_profile.go

```diff
@@ -0,0 +1,87 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmLogProfile() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmLogProfileRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"storage_account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"service_bus_rule_id": {
```

Comment:
can we make this `servicebus_rule_id` to match the other resources/Azure branding?